packages/torbrowser.rb

```diff
@@ -3,23 +3,34 @@
 class Torbrowser < Package
   description "'The Onion Router' browser"
   homepage 'https://www.torproject.org/'
-  @_ver = '11.5.1'
+  @_ver = '11.5.7'
   version @_ver
   license 'BSD, custom, MPL-2.0 and MIT'
   compatibility 'x86_64'
   source_url "https://dist.torproject.org/torbrowser/#{@_ver}/tor-browser-linux64-#{@_ver}_en-US.tar.xz"
-  source_sha256 '2e0cefa6d4284c71a7816e310d935f9b9b5e4a3a408dc53330a0db0474489e8a'
+  source_sha256 '2b9d13f457bab4cb8fd49e607f02bd7ffdb965e6a443dbec262e0838f6ba7cc9'
 
   depends_on 'gtk3'
   depends_on 'sommelier'
 
   def self.build
-    @tor = <<~EOF
-      #!/bin/sh -e
+    tor = <<~EOF
+      #!/bin/bash
+      SCALE=1
+      RESOLUTION=$(xdpyinfo | awk '/dimensions:/ { print $2 }' | cut -d'x' -f1)
```

Comment:
Nice. But do we want to add this sort of logic to sommelier?

Reply:
This logic is specific to gtk3 so maybe `/usr/local/etc/gtk-3.0/settings.ini`?

Reply:
Actually, it might make sense to include it in the X11 config.